src/guides/bioinformatics/exporting-data-for-auspice.md

```diff
@@ -0,0 +1,149 @@
+## How data is exported by Augur for visualisation in Auspice
+
+This page is intended to describe which data underpins the different parts of an Auspice visualisation, and how this data can be produced via Augur. It is not a comprehensive guide to how to generate individual pieces of data nor how to interpret them in Auspice. 
+
+### Auspice components
+
+It’s helpful to start in auspice and then work backwards to augur. The following (annotated) screenshot shows the components of a typical visualisation, which is based on a dataset JSON (sometimes called an Auspice JSON). 
+
+Generally, data is defined on individual tree nodes in this dataset (via `node.node_attrs: Dictionary` or `node.branch_attrs: Dictionary` ) or within `meta: Dictionary` . We will refer to these below, however the best place to really understand the data structures is  [the JSON schema](https://github.com/nextstrain/augur/blob/master/augur/data/schema-export-v2.json).  
+
+This guide is based on Auspice version 2.37.3. We will endeavour to keep this up-to-date however if you are using a newer version of auspice it may pay to check the [changelog](https://github.com/nextstrain/auspice/blob/master/CHANGELOG.md) for any salient changes.
+
+![](../../images/auspice-components.png)
+
+**[a1] Colorings**
+
+The available colorings are defined by  `meta.colorings: Array<dict>`, where each colouring specifies a key which accesses the relevant data in the `node_attrs` [1]. As well as a key, a title can be specified (which is what the user will see), as well as information about the scale used.
+
+**[a2] sidebar data filtering**
+
+Currently the  `node_attrs` available for filtering are the union of those defined in `meta.filters: Array<string>` , strain names (`node.name: String`) and mutations (`branch_attrs.mutations` ).
+
+> We will hopefully change this shortly to include all data defined in a `node.node_attrs`.
```

Comment:
Using rST would allow this callout to be a proper note directive instead of an indented blockquote.

Reply:
I've italicised it for now...